cli/templatepush.go

```diff
@@ -233,6 +236,12 @@ func (r *RootCmd) templatePush() *clibase.Cmd {
 			Description: "Always prompt all parameters. Does not pull parameter values from active template version.",
 			Value:       clibase.BoolOf(&alwaysPrompt),
 		},
+		{
+			Flag:        "make-active",
```

Comment:
Let's make this `activate`, and the variables too.

Reply:
When I run make, I get errors that seem unrelated to this change:

<details>
<summary>Errors
</summary>

```
# github.com/cenkalti/backoff/v4
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:10:26: syntax error: unexpected any, expecting ]
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:10:38: method has no receiver
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:16:53: syntax error: unexpected [ after top level declaration
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:17:16: method has no receiver
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:17:16: syntax error: unexpected (, expecting name or (
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:42:19: syntax error: unexpected [, expecting (
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:53:25: syntax error: unexpected [, expecting (
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:66:33: syntax error: unexpected [, expecting (
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:70:19: syntax error: unexpected [, expecting (
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:76:2: syntax error: non-declaration statement outside function body
../../go/pkg/mod/github.com/cenkalti/backoff/v4@v4.2.0/retry.go:76:2: too many errors
note: module requires Go 1.18
# github.com/coder/coder/cryptorand
cryptorand/slices.go:9:6: missing function body
cryptorand/slices.go:9:13: syntax error: unexpected [, expecting (
note: module requires Go 1.20
Makefile:473: recipe for target 'coderd/database/dump.sql' failed
make: *** [coderd/database/dump.sql] Error 2
```

</details>